proto/zetachain/zetacore/observer/operational.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package zetachain.zetacore.observer;
+
+option go_package = "github.com/zeta-chain/node/x/observer/types";
+
+message OperationalFlags {
+  // Height for a coordinated zetaclient restart.
+  // Will be ignored if missed.
+  int64 restart_height = 1;
```

Comment:
Consider using unit64

Reply:
It seems we use int64 for most other heights in the observer package so I would prefer to stay consistent.

Examples:
- `keygen_zeta_height`
- `finalizedZetaHeight`
- `keyGenZetaHeight`

Reply:
We use uint64 as well, for example 
`GasPrice` [For external heights]
`LastBlockHeight`

The cosmos block height is int64; I guess it might make sense to use int64, where we are planning to compare against the cosmos height, Which I don't think is the case here; I don't believe this height can ever be negative.

Reply:
We use int64 for block height in general, even in geth, it's an antipattern but better to maintain consistency and avoiding conversion with overflow risk, this value is set through admin prop so low risk

Reply:
I agree with the fact that we should use one type if choosing int64 it would be good to have an issue to refactor the ones I mentioned to int64 at some point

Reply:
>  issue to refactor the ones I mentioned to int64 at some point

breaking changes are 10x worse than inconsistent api design imo.